posthog/api/team.py

```diff
@@ -459,6 +459,16 @@ def validate_session_replay_ai_summary_config(value: dict | None) -> dict | None
 
         return value
 
+    def validate_app_urls(self, value: list[str | None] | None) -> list[str] | None:
+        if value is None:
+            return value
+        return [url for url in value if url is not None]
```

Comment:
i think in python just `if url` is enough
since then we also validate for empty strings
(or at least i remember that python is much more sensible about this type of thing than JS)